docs/versioning/how-to-view-versions.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+nav_order: 5
+title: How to view versions  
+has_children: false
+parent: Versioning
+---
+# How to view versions
+
+
+## Viewing version info in the Registry
+
+
+### Step 1. Navigate to Versioning tab in the item record in the Registry
+
+- Ensure you're logged in.
```

Comment:
This is important, but only mentioned on this page - its required for every process and maybe not worth stating at this point?

Reply:
Suggests we need a Pre-requisites section before each of these how-to's

Reply:
done